ui/src/App.js

```diff
@@ -14,6 +14,17 @@ import FacetsGrid from "components/facets/FacetsGrid";
 import FieldSearch from "components/FieldSearch";
 import Header from "components/Header";
 
+const Disclaimer = (
+  <div style={{margin: '20px'}}>
+    This dataset is publicly available for anyone to use under the terms
+    provided by the dataset source <a href="http://www.internationalgenome.org/data">
```

Comment:
Put parenthesis around the link, like https://cloud.google.com/genomics/docs/public-datasets/1000-genomes

Reply:
Done.